compiler-core/src/javascript/typescript.rs

```diff
@@ -294,41 +294,43 @@ impl<'a> TypeScriptGenerator<'a> {
         match statement {
             Definition::TypeAlias(TypeAlias {
                 alias,
-                public,
+                publicity,
                 type_,
                 ..
-            }) if *public => vec![self.type_alias(alias, type_)],
+            }) if !publicity.is_internal() => vec![self.type_alias(alias, type_)],
```

Comment:
Why is this rendered for public and private but not internal?

Reply:
Oh that's a typo! It was meant to be `!publicity.is_private()`

Reply:
oh no! 😁